### ISSUE TITLE
Paint Brush

### ISSUE THREAD

**Issue by Gingerston** [zakmasood/BricksVRModding#1]
A paintbrush you can touch to the color area and apply to blocks that have been placed, would speed up building things like signs or more detailed builds.

**Comment by ATXLtheAxolotl** [comment]
I could maybe make a mod for this, I think the difficult part though would be selecting colors.

**Comment by ATXLtheAxolotl** [comment]
Actually no I could just get the color from the color picker.

**Comment by zakmasood** [comment]
Yep, easiest way would be to talk the value from the color picker, and make a brush model. Have a raycast come out of the brush
and affect any brick it comes into contact to, setting the brick to the one selected in the color picker
